docs/components/console/manage-organization/usage-history.md

```diff
@@ -5,11 +5,17 @@ description: "Let's observe our workflow usage."
 ---
 
 :::note
-This setting is only visible during the **Trial Phase** for owners and admins.
+The usage history is visible only to owners and admins.
```

Comment:
well, the `Usage` tab is only available in non-paying salesplans - so this is is not too cowrecked i think?
for paying orgs it is in `Billing`